### ISSUE TITLE
Tests failing on Ubuntu 14.04 LTS

### ISSUE THREAD

**Issue by muhammaddadu** [clout-stack/clout-services-child#1]
The error seems to show a command is not found in the following test output

``` bash
    1) create docker container w/ NodeJS application


  0 passing (25s)
  1 failing

  1) services test create docker container w/ NodeJS application:

      Uncaught AssertionError: child process exited with code 1
The command '/bin/sh -c apt-get update &&     apt-get -y install curl &&     curl -sL https://deb.nodesource.com/setup | sudo bash - &&     apt-get -y install python build-essential nodejs' returned a non-zero code: 127

      + expected - actual

      -false
      +true

    at Assertion.fail (node_modules/should/lib/assertion.js:92:17)
    at Assertion.Object.defineProperty.value (node_modules/should/lib/assertion.js:164:19)
    at Request._callback (test/services_test.js:53:27)
    at Request.self.callback (node_modules/request/request.js:200:22)
    at Request.<anonymous> (node_modules/request/request.js:1067:10)
    at IncomingMessage.<anonymous> (node_modules/request/request.js:988:12)
    at endReadableNT (_stream_readable.js:905:12)
```

**Comment by muhammaddadu** [comment]
After the commit [8ee194897be8fe5877e0f56e77f733f66ec60449](https://github.com/clout-stack/clout-services-child/commit/8ee194897be8fe5877e0f56e77f733f66ec60449) the error comes down to

```
RUN curl -sL https://deb.nodesource.com/setup | sudo bash
```

as seen in the error

``` bash
Uncaught AssertionError: child process exited with code 1
The command '/bin/sh -c curl -sL https://deb.nodesource.com/setup | sudo bash' returned a non-zero code: 127
```

**Comment by muhammaddadu** [comment]
Issue has been resolved by the commit [da75ff69f57efd12438c35bc6b0dd12f20da644f](https://github.com/clout-stack/clout-services-child/commit/da75ff69f57efd12438c35bc6b0dd12f20da644f).

The output log now looks like

``` bash
PORT=8083 grunt test
Running "mochaTest:test" (mochaTest) task


  services test
http server started on port 8083
http://127.0.0.1:8083/api/dev/tester/service?application={"name":"tester","id":"tester"}
    ✓ create docker container w/ NodeJS application (52907ms)


  1 passing (53s)


Done.
```

checking the docker running containers

``` bash
docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS                     NAMES
5bfa93403755        tester/tester       "nodemon ."         16 seconds ago      Up 15 seconds       0.0.0.0:32769->8080/tcp   tester-tester.n1
```
